token/src/contract.rs

```diff
@@ -155,48 +165,6 @@ impl TokenTrait for Token {
         event::burn(&e, from, amount)
     }
 
-    fn clawback(e: Env, from: Address, amount: i128) {
-        check_nonnegative_amount(amount);
-        let admin = read_administrator(&e);
-        admin.require_auth();
-
-        e.storage().instance().bump(INSTANCE_BUMP_AMOUNT);
-
-        spend_balance(&e, from.clone(), amount);
-        event::clawback(&e, admin, from, amount);
-    }
-
-    fn set_authorized(e: Env, id: Address, authorize: bool) {
-        let admin = read_administrator(&e);
-        admin.require_auth();
-
-        e.storage().instance().bump(INSTANCE_BUMP_AMOUNT);
-
-        write_authorization(&e, id.clone(), authorize);
-        event::set_authorized(&e, admin, id, authorize);
-    }
-
-    fn mint(e: Env, to: Address, amount: i128) {
-        check_nonnegative_amount(amount);
-        let admin = read_administrator(&e);
-        admin.require_auth();
-
-        e.storage().instance().bump(INSTANCE_BUMP_AMOUNT);
-
-        receive_balance(&e, to.clone(), amount);
-        event::mint(&e, admin, to, amount);
-    }
-
-    fn set_admin(e: Env, new_admin: Address) {
-        let admin = read_administrator(&e);
-        admin.require_auth();
-
-        e.storage().instance().bump(INSTANCE_BUMP_AMOUNT);
-
-        write_administrator(&e, &new_admin);
-        event::set_admin(&e, admin, new_admin);
-    }
```

Comment:
A nice side-effect of this change is that the example contract is clearer about which functions it exposes are part of the token interface and which are its own functions that provide functionality that isn't described by the standard token interface.